### ISSUE TITLE
Finalise Sentiment analysis approach.

### ISSUE THREAD

**Issue by hauntedcupoftea** [projectayre/ayre#10]
It's getting harder to maintain relevance to soft computing. None of the main ML libraries support fuzzy neural network or logic. Anything done with respect to that will be novel and experimental, but exponentially more difficult as a result.

**Comment by hauntedcupoftea** [comment]
dataloader isn't wokring for me, throws a new error each time. very confused.

**Comment by hauntedcupoftea** [comment]
someone owes me a drink for figuring out that entire torch dataset shenanigan

**Comment by 7Zenox** [comment]
#4 has been closed. We are exploring #15 as a new approach for sentiment analysis.

**Comment by hauntedcupoftea** [comment]
We will be working on this as follows:
A multi-channel U net type model where every label from the segmentation task gets its own layer.
This layer can be used in multiple ways:
- Convert into a projection map for annotation using simple matrix operations
- Use each layer directly as an attention mask for the VQA model, to ensure every segment gets its own self attention.
LGTM!
